app/controllers/posts_controller.rb

```diff
@@ -9,7 +9,7 @@ def around
 
   def live_feed(postId)
     post = Post.find_by_id(postId)
-    return Post.order(taken_at: :desc).where("latitude > ?" , post.latitude - 0.001).where("latitude < ?" , post.latitude + 0.001).where("longitude > ?" , post.longitude - 0.001).where("longitude < ?" , post.longitude + 0.001).where("taken_at > ?", post.taken_at - 1.hour)
+    return Post.order(taken_at: :desc).where("latitude > ?" , post.latitude - 0.001).where("latitude < ?" , post.latitude + 0.001).where("longitude > ?" , post.longitude - 0.001).where("longitude < ?" , post.longitude + 0.001).where("taken_at > ?", post.taken_at - 1.hour).includes(:comments)
```

Comment:
perhaps its a good idea to create a couple of constants:
```
SEARCH_RADIUS = 0.001
SEARCH_TIME = 1.hour
```
So you can change it easily at the top of your controller and reuse it